tests/components/SearchBar.test.tsx

```diff
@@ -147,34 +147,34 @@ describe('SearchBar', () => {
         .mockResolvedValue(mockedUniversalAutocompleteResult);
     });
 
-    it('displays vertical links as part of the query suggestions', async () => {
+    it('does not display vertical links on default', async () => {
```

Comment:
I think it would have been less code changes to flip the props in the test instead of the order of the tests but this also works!